report_thesis/src/sections/proposed_approach/model_selection.tex

```diff
@@ -0,0 +1,103 @@
+
+While these preprocessing techniques are not an exhaustive list, they represent a diverse set of methods.
+Techniques such as feature selection were not considered, in order to limit the scope of the study and due to time constraints.
+
+We also had several requirements for the selection of models.
+The models selected for experimentation had to be diverse to ensure that we had enough breadth in our results to make informed decisions about which models should be included in the final stacking ensemble pipeline.
+Additionally, the models had to be suitable for regression tasks and if no research was available in the context of \gls{libs} data, they had to be models that have shown promise in other domains.
+Our literature review found that a variety of models fit this criteria.
+For example, \citet{andersonPostlandingMajorElement2022} demonstrated that that models such as \gls{gbr}, \gls{pls}, \gls{lasso}, \gls{rf} were each effective at predicting different major oxides.
+Additionaly, \citet{svrforlibs} demonstrated in their work that \gls{svr} outperforms \gls{plsr} in predicting \ce{Si}, \ce{Ca}, \ce{Mg}, \ce{Fe} and \ce{Al}.
+As a result, we have included \gls{gbr}, \gls{pls}, \gls{lasso}, \gls{rf}, and \gls{svr} in our experiments.
+
+In the neural network domain, \cite{ann_libs_soil_analysis} showed that on \gls{libs} data their 3-layer \gls{ann} relative error of prediction for \ce{Ca}, \ce{Fe}, \ce{Al} was below 20\%.
+Likewise, \citet{yangConvolutionalNeuralNetwork2022} showed that \gls{cnn} outperformed methods such as \gls{logreg}, \gls{svm} and linear discriminant analysis at correctly classifying twelve different types of rocks based on \gls{libs} data. 
+While this example for \gls{cnn} is a classification task, \gls{cnn} can be adjusted for regression tasks by changing the loss function and output layer.
+Based on these factors, we decided to include \gls{ann} and \gls{cnn} in our experiments to further increase the diversity of our model selection.
+
+To further bolster our selection pool, we chose to include models that were in the same family of models as those that showed promise in the literature.
+
+\gls{xgboost} was included as an option based on its promising accuracy in various settings.
```

Comment:
"as an option"? What do you mean?

Reply:
Option is a bad choice of words. I suggest this instead.
```suggestion
\gls{xgboost} was included in our experiments based on its promising accuracy in various settings.
```